spec/features/admin_creates_trip_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+feature "Admin features" do
+  describe "admin creates a trip" do
+    before :each do
+      User.create(username: "Alex", password: "password", role: 1)
+      visit root_url
+      click_link "Login"
+      fill_in("Username", with: "Alex")
+      fill_in("Password", with: "password")
+      click_button("Login")
+
+    end
+
+    context "admin dashboard" do
+      it "has a link to create a trip" do
+        visit admin_dashboard_path
+        click_link "Create a New Trip"
+
+        expect(current_path).to eq(new_admin_trip_path)
+        within("h2") { expect(page).to have_content("Create a New Trip") }
+      end
+    end
+
+    context "create-trip page" do
+      it "creates a trip with valid params" do
+        Destination.create(continent: "Asia", description: "description",
+                    lat: 31.8369, long: -77.6847, image_url: "asia.jpg" )
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside parentheses detected.